plancton/__init__.py

```diff
@@ -217,11 +217,17 @@ def _read_conf(self):
 
   # Set up monitoring target.
   def _influxdb_setup(self):
+    self.streamers = []
     if not self.conf["influxdb_url"]:
-      self.streamer = lambda *x,**y: True
+      self.streamers.append(lambda *x,**y: True)
       return
-    baseurl,db = self.conf["influxdb_url"].split("#")
-    self.streamer = InfluxDBStreamer(baseurl=baseurl, database=db)
+    if isinstance(self.conf["influxdb_url"], list):
```

Comment:
You are doing the same thing twice, some code is duplicated. Instead, for being backwards-compatible and supporting both lists and a single value for `influxdb_url`, do the following:

* If the value is _not_ a list, then convert it to a list with one item: `a --> [a]`
* Process the list item by item